README.md

```diff
@@ -1,2 +1,75 @@
-# sas-education
-A collection of hands-on instructions, examples, data, and tutorials provided by SAS Education to support your SAS training journey.
+<!--
+Use this template to structure your project's README.md file.
+Not all sections are required, though we recommend including as many as possible.
+
+* Project_Name (required)
+* Overview (required)
+  * What's new (optional)
+  * Prerequisites (optional)
+* Installation (required)
+  * Getting Started (optional)
+  * Running (optional)
+  * Examples (optional)
+  * Troubleshooting (optional)
+* Contributing (required)
+* License (required)
+* Additional Resources (required)
+
+Consider using a table of contents to make lengthy and complex README files easier to navigate.
+-->
+
+# SAS Education
+
+## Overview
+This reppository contains a collection of hands-on instructions, examples, data, and tutorials provided by SAS Education to support your SAS training journey.
+
+## Installation
+<!--
+Provide step-by-step instructions for installing your software project.
+Use subtopics and screenshots as appropriate.
+-->
+
+### Getting Started
+<!--
+Provide users with initial steps for getting started using your project after they have installed it.
+This is a good place to include screenshots, animated GIFs, or short example videos.
+-->
+
+### Running
+<!--
+Provide users with steps for running your project after they have installed it.
+This is a good place to include screenshots, [asciinema](https://asciinema.org/) recordings, or short usage videos.
+-->
+
+### Examples
+<!--
+Provide additional examples of using the software, or point to further documentation. 
+Make learning and using your project as easy as possible!
+-->
+
+### Troubleshooting
+<!--
+Provide workarounds and solutions to known problems.
+Organize troubleshooting information using subtopics, as appropriate.
+-->
```

Comment:
@nicrobert: Could you please help me complete this section with instructions that could help users get started using the resources?